kata-containers-image/LICENSE

```diff
@@ -0,0 +1,2109 @@
+=========================================================================
+This package contains next projects with following licenses
```

Comment:
I dont know if this is really needed ? 
IF that is the case the osbuilder should provde with the list of packages it added to the image?